### PR TITLE
IDE-3909 delete server and runtime when delete workspace project

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/LiferayWorkspaceProjectDeleteParticipant.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+		if ((bundlesLocation == null) || FileUtil.notExists(bundlesLocation)) {
+			return change;
+		}
+
+		IServer[] servers = ServerCore.getServers();
+
+		Stream<IServer> serverStream = Stream.of(servers);
+
+		serverStream.filter(server -> server != null)
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/LiferayWorkspaceProjectDeleteParticipant.java]: Sf

**Comment by jtydhr88** [comment]
Please change the comment description to about add the Maven Liferay Workspace and DeleteParticipte

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/LiferayWorkspaceProjectDeleteParticipant.java (hunk 1 above)
Comment (simonjhy) [reply tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/LiferayWorkspaceProjectDeleteParticipant.java]: I have used the latest source format tool yesterday. let me check what's wrong

Comment (simonjhy) [reply tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/LiferayWorkspaceProjectDeleteParticipant.java]: I used 1.0.648 version sf

**Pull request closed without merging**